com.obsproject.Studio.V4L2Sink.json

```
@@ -0,0 +1,37 @@
+{
+  "id": "com.obsproject.Studio.V4L2Sink",
+  "branch": "stable",
+  "runtime": "com.obsproject.Studio",
+  "runtime-version": "stable",
+  "sdk": "org.kde.Sdk//5.14",
+  "build-extension": true,
+  "separate-locales": false,
+  "appstream-compose": false,
+  "modules": [
+    {
+      "name": "v4l2sink",
+      "buildsystem": "cmake",
+      "no-make-install": true,
+      "config-opts": [
+        "-DLIBOBS_INCLUDE_DIR=./obs-source/libobs"
+      ],
+      "post-install": [
+        "install -Dm644 v4l2sink.so /app/lib/v4l2sink/v4l2sink.so"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/CatxFish/obs-v4l2sink/archive/1ec3c8ada0e1040d867ce567f177be55cd278378.tar.gz",
+          "sha256": "381f9a4346aa5d926a07116160957b38fb35c89173f441180b7c25b84c9463e5"
+        },
+        {
+          "type": "archive",
+          "//": "It needs a copy of the source for some cmake files, its easier to just download again",
```

```suggestion
          "//": "It needs a copy of the source for some cmake files",
```

That specific comment was related to plugins in-tree